databricks.yml

```
@@ -0,0 +1,94 @@
+              - "--git_sha"
+              - ${var.git_sha}
+          libraries:
+            - whl: ./dist/*.whl
+
+targets:
+  dev:
+    mode: development
+    default: true
+    workspace:
+      host: https://adb-955835127093591.11.azuredatabricks.net
+      root_path: /Workspace/Users/k.patyk@paulaschoice-eu.com/.bundle/${bundle.target}/${bundle.name}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)